install-dev/upgrade/php/ps_1750_update_module_tabs.php

```diff
@@ -29,15 +29,34 @@
  */
 function ps_1750_update_module_tabs()
 {
-    // Add new sub menus for modules
+    // STEP 1: Add new sub menus for modules
     $moduleTabsToBeAdded = array(
         'AdminModulesUpdates' => 'en:Updates|fr:Mises à jour|es:Actualizaciones|de:Aktualisierung|it:Aggiornamenti',
     );
 
     include_once('add_new_tab.php');
     foreach ($moduleTabsToBeAdded as $className => $translations) {
         add_new_tab_17($className, $translations, 0, false, 'AdminModulesSf');
+        Db::getInstance()->execute('UPDATE `'._DB_PREFIX_.'tab` SET `active`= 1 WHERE `class_name` = "' . $className . '"');
     }
 
-    Db::getInstance()->execute('UPDATE `'._DB_PREFIX_.'tab` SET `active`=1 WHERE `class_name` = "AdminModulesUpdates"');
+    
+    // STEP 2: Rename Notifications as Alerts
+    $adminModulesNotificationsTabId = Db::getInstance()->getValue('SELECT id_tab FROM '._DB_PREFIX_.'tab WHERE class_name = "AdminModulesNotifications"');
+    if (empty($adminModulesNotificationsTabId)) {
+        return;
+    }
+    
+    include_once('clean_tabs_15.php');
+    renameTab(
+        $adminModulesNotificationsTabId,
+        [
+            'fr' => 'Alertes',
```

Comment:
stupid question: what will happens for others languages?

Reply:
Heh, they keep using the old name...

Reply:
sad :'(

Reply:
This is what I talked about this morning, renaming tabs should be done in another way. Currently I'm just following the same method since PrestaShop 1.5.